.github/workflows/ScreenShotTest.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  screenshot-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Download Screenshots
+        id: download-artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: UpdateScreenshots.yml
+          name: screenshots
+          path: preview-screenshots/src/test/snapshots/images
+
+      - name: Run screenshot tests
+        id: verifyPaparazziDebug
+        continue-on-error: true
+        run: ./gradlew preview-screenshots:verifyPaparazziDebug
+
+      - name: Error Screenshot tests
+        continue-on-error: true
```

Comment:
This may not be necessary, because the `verifyPaparazziDebug` step specifies `continue-on-error: true`, so it will always succeed.

Reply:
I forgot to delete this one:sweat_smile: